<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(103, 162, 236)</string>
    </property>
    <property name="text">
     <string>Load Network</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(50, 203, 255)</string>
    </property>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 165, 224)</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 165, 224)</string>
     </property>
     <property name="text">
      <string>Find Minimum cut 
from Source A to Sink B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 165, 224)</string>
     </property>
     <property name="text">
      <string>Compute maxflow on 
Y paths flowing over X 
bottleneck links 
(PUV's) | Peak traffic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 165, 224)</string>
     </property>
     <property name="text">
      <string>Compute maxflow 
on Y paths flowing over X 
bottleneck links 
(PUV's) | No traffic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 165, 224)</string>
     </property>
     <property name="text">
      <string>Compute maxflow
from Source A
to Sink B (prompts) 
|No traffic on network </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="text">
     <string>Load image here</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(103, 162, 236)</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(103, 162, 236)</string>
    </property>
    <property name="text">
     <string>Create Graph</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(103, 162, 236)</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>191</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(50, 203, 255)</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 165, 224)</string>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 6</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 7</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 8</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 9</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image 10</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionView_Network">
   <property name="text">
    <string>View Network</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
